.travis.yml

```diff
@@ -183,7 +183,7 @@ jobs:
           --log-template "totals-complete"
           --log-template "errors"
           --
-          || { printf "\nDiff code coverage check failed. To view coverage report, run 'mvn clean test jacoco:report' and open 'target/site/jacoco/index.html'\n" && false; }
+          || { printf "\n\n****FAILED****\nDiff code coverage check failed. To view coverage report, run 'mvn clean test jacoco:report' and open 'target/site/jacoco/index.html'\nFor more instructions on how to run code coverage locally, follow instructions here - https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md#running-code-coverage-locally\n\n" && false; }
```

Comment:
nit: "For more instructions on how to run code coverage locally, follow instructions.." reads sort of funny, maybe one of the 'instructions' should be replaced with 'details' or some other word?